testsuite/dos/http_test/README.md

```diff
@@ -0,0 +1,10 @@
+# HTTP Tester 
+creates an axum server behind an haproxy to test http configs for haproxy
+
+## Server
+1. cargo build --release
+2. cd docker
+3. docker compose up
+
+## Test
```

Comment:
What is this testing? 

Also can this run in ci? (otherwise if we rely on manual testing people will definitely accidentally break this)

Reply:
This is for developer testing, when you are making changes and want to test the impact. 
This is not a CI/CD test.